### PR TITLE
II-4679 Edited MySQL agent for SQLServer

### PR DESCRIPTION
**Pull request by mellv**

### CODE REVIEW

File: mssql_log_collector/cron-mysql-log.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+AGENT="mssql-log"
```
Comment (gnegri) [inline mssql_log_collector/cron-mysql-log.sh]: rename this file to 'cron-mssql-log.sh' to reflect the Agent type.

File: mssql_log_collector/get_logs_mssql.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        print("config.ini file is missing")
+    return mssql_config_map, if_config_map
+
+
+'''
+Connect to the database by the given credentials
+'''
+def connect_to_database(host, database, user, password):
+    try:
+        connection = pyodbc.connect('DRIVER={ODBC Driver 17 for SQL Server};SERVER='+host+';DATABASE='+database+';UID='+user+';PWD='+ password)
```
Comment (gnegri) [inline mssql_log_collector/get_logs_mssql.py]: Make the DRIVER= value configurable.

File: mssql_log_collector/pip-mysql-log.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```
Comment (gnegri) [inline mssql_log_collector/pip-mysql-log.sh]: rename to 'pip-mssql-log.sh'

File: mssql_log_collector/cron-mysql-log.sh (hunk 1 above)
Comment (mellv) [reply mssql_log_collector/cron-mysql-log.sh]: renamed

File: mssql_log_collector/get_logs_mssql.py (hunk 2 above)
Comment (mellv) [reply mssql_log_collector/get_logs_mssql.py]: done

File: mssql_log_collector/pip-mysql-log.sh (hunk 3 above)
Comment (mellv) [reply mssql_log_collector/pip-mysql-log.sh]: renamed

**Pull request merged**